test/unit/core/dom/layout/test-viewport-observer.js

```diff
@@ -124,5 +125,22 @@ describes.sandboxed('DOM - layout - Viewport Observer', {}, (env) => {
 
       expect(spy).not.called;
     });
+
+    it('can observe multiple elements', () => {
```

Comment:
Done. PTAL.

Reply:
You can remove this test then :P

Reply:
Done. Original intention of leaving it was because they were testing diff methods. But since `observeWithSharedInOb` is a wrapper around `observeIntersections`, makes sense to remove it.

Reply:
Ohhh I see! So it could make sense to change the method to test `observeIntersections` Feel free to punt on that until later when migrating